source/keyLabels.py

```diff
@@ -164,4 +164,11 @@
 }
 
 def getKeyCombinationLabel(keyCombination):
-	return "+".join(localizedKeyLabels[key.lower()] for key in keyCombination.split("+"))
+	"""
+	Fetches a localized label for a combination of multiple keys.
+	@param keyCombination: The key combination identifier to get the localized label for, usually plus-separated key identifiers.
+	@type keyCombination: string
+	@returns: A localized key combination
+	@rtype: string
+	"""
+	return "+".join(localizedKeyLabels[key.lower()] if key.lower() in localizedKeyLabels.keys() else key.lower() for key in keyCombination.split("+"))

```

Comment:
1. This certainly demonstrates the power of comprehensions :), but it does mean you end up calling `key.lower()` twice per key.
2. Calling `dict.keys` generates a list of keys you then have to search, which is unnecessary here. For future reference, use `key in dict`, not `key in dict.keys()`.
3. You can eliminate the existence check by using `dict.get` with a default.

While premature optimisation is the root of all evil, I think it's worth eliminating unnecessary calls, especially if it makes things more readable. Perhaps something like this:

```
keys = keyCombination.lower().split("+")
return "+".join(localizedKeyLabels.get(key, key) for key in keys)
```